docs/developer-setup/README.md

```diff
@@ -697,13 +498,28 @@ Each workspace contains recommended extensions and settings for VSCode developme
 - Format - formats the code with clang-format
 - Lint - runs clang-tidy
 - Lint Fix - runs clang-tidy and fixes issues
-- Dependency graph - shows the third party library dependency graph
+- Dependency Graph - shows the third party library dependency graph
 
 To run a task:
 
 - `Ctrl + Shift + B` and select the task, e.g. `Build`
 - Select the build type and compiler (if applicable)
 
+### Debugging
+
+Windows and Linux versions of `launch.json` are provided in the `.vscode` folder.
+
+* On Windows copy `launch.json.windows` and rename it to `launch.json`.
```

Comment:
The symbolic link worked here for me. The user wouldn't have to re-copy if we edit a launch.json.*. It's a bit more complex, but the user already has to do the exact same thing for the python3.exe->python.exe symlink.